lib/generator._js

```diff
@@ -460,29 +402,32 @@ ScriptGenerator.prototype.generateDotNetDeploymentScript = function (templateFil
     .replace(/{MSBuildArgumentsForInPlace}/g, options.msbuildArgumentsForInPlace || "")
     .replace(/{SolutionPath}/g, this.solutionPath || "")
     .replace(/{SolutionDir}/g, solutionDir)
-    .replace(/{SitePath}/g, fixPathSeperatorToWindows(this.sitePath));
+    .replace(/{SitePath}/g, fixPathSeparatorToWindows(this.sitePath));
 
   this.writeDeploymentFiles(templateContent, _);
 };
 
 ScriptGenerator.prototype.generateAspNetCoreScript = function (templateFileName, options, _) {
   argNotNull(templateFileName, 'templateFileName');
 
-  var fixedAspNetCoreProject = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.aspNetCoreProject) : fixPathSeperatorToUnix(options.aspNetCoreProject);
-  var fixedRestore = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.restore) : fixPathSeperatorToUnix(options.restore);
-  var fixedSolutionPath = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.solutionPath) : fixPathSeperatorToUnix(options.solutionPath);
-  
+  if (this.scriptType === ScriptType.bash) {
```

Comment:
For Powershell as well? Honestly not sure. I think maybe it handles Unix and mixed path separators, but not positive.